arbos/storage/storage.go

```diff
@@ -450,6 +450,10 @@ func (sbbi *StorageBackedBigInt) Set(val *big.Int) error {
 	return sbbi.StorageSlot.Set(common.BigToHash(val))
 }
 
+func (sbbi *StorageBackedBigInt) SetByUint(val uint64) error {
```

Comment:
For safety, this should check for overflow, to make sure the value isn't larger than MaxInt64.

Reply:
Are you sure? This is for big int. The other setter above doesn’t have a check so we’d need one there too if so